File: 1.py
#외부에서 내부변수를 조회하는 것을 막으려면 변수 앞에 "__"를 넣음
class C(object):
    def __init__(self, v):
        self.__value = v
    def show(self):
        print(self.__value)

c1 = C(10)
# print(c1.value)
c1.show()

File: 9.Class/1.rb
# ruby 내부 변수를 외부에서 조회하려면 attr_reader, attr_wirter, attr_accesor를 사용
class C
    # attr_reader :value
    # attr_writer :value
    attr_accessor :value
    def initialize(v)
        @value = v
    end
    def show()
        p @value
    end
end


c1=C.new(10)
p c1.value
c1.value = 20
p c1.value